pom.xml

```diff
@@ -32,6 +37,21 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>${org.mapstruct.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
```

Comment:
I think @eazaretzky asked to not use lombok

Reply:
This was discussed with @agustinb4 in order to avoid excessive boilerplate and leverage the Builder pattern, but if you guys think this is an incorrect approach I'm open to suggestions.